### DISCUSSION TITLE
github.com does not work properly

### DISCUSSION THREAD

**Discussion by am-trouzine** [community/community#110916] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

The website does not display properly. I am using Chrome/FireFox on Android 4.4.2. This has started a long time ago. I mainly use the site to download binaries and display codes.
The pages look very plane. Most of the links are either actionless or reload the page. The menu under "Assets" never display and the circle turns around forever; displaying the folders requires switching to pc mode but the folders content is always empty; trying to click on a file redirects to main directory...
I hope that the maintainers are made aware of this issue.

**Comment by gil-son** [comment, marked as answer]
Hello,

try to access this link: https://am-trouzine.github.io/my-git-site/

![link_prove](https://github.com/community/community/assets/72712095/a6641dde-157b-463f-a728-58333e679dda)


If I suppoted you consider to mark my answer

**Comment by am-trouzine** [comment]
Thank you @Gilson for your reply. The issue is actually about the github.com
site. Here's how it looks on Chrome:
https://drive.google.com/file/d/1rrg3abj0IsyqjxDRfJ0S49iCMtK90Utr/view?usp=sharing


Le lundi 4 mars 2024, Gilson Vieira de Souza ***@***.***> a
écrit :

**Reply by gil-son** [reply]
This is my view when I open your page in the Chrome

Chrome:

![chrome](https://github.com/community/community/assets/72712095/8da2f54c-f2bb-4dd9-84c5-91d9f213223f)


I use Chrome from Linux
